tests/trainer/properties/test_get_model.py

```diff
@@ -84,8 +83,7 @@ def test_get_model_gpu(tmpdir):
 
 
 @RunIf(min_gpus=1, skip_windows=True)
-@DDPLauncher.run("--accelerator [accelerator]", max_epochs=["1"], accelerator=["ddp", "ddp_spawn"])
-def test_get_model_ddp_gpu(tmpdir, args=None):
+def test_get_model_ddp_gpu(tmpdir):
```

Comment:
Why don't we need the DDPLauncher here any more?

Reply:
We never needed it :)

Reply:
it doesn't even do anything.
this test can be deleted, now that not even the name has a meaning anymore